customer.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/main.css">
+    <link rel="stylesheet" href="./assets/css/customer.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <script src="https://kit.fontawesome.com/34321f5008.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Lobster&family=Open+Sans&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Fashion-Customer page</title>
+</head>
+<body>
+    <header class="header cs">
+        <a href="index.html" class="brandName">Fashion</a>
+        <div class="search-container">
+         <form >
+           <input type="text" placeholder="Search.." name="search">
+           <button type="submit"><i class="fa fa-search"></i></button>
+         </form>
+        </div>
+        <button id="customerbtn">Customer</button>
+        <button id="sellerbtn">Seller</button>
+        
+    </header>
+
+    <div class="navmenu">
+        <a href="#">Home</a>
```

Comment:
since this link for `home` page, put in the `href="index.html"` to redirect you to your landing page.